### DISCUSSION TITLE
AMARI UE Simbox

### DISCUSSION THREAD

**Discussion by Lusani1** [srsran/srsRAN_Project#452] [General Help]
Dear All,
I would like to do some stress testing and energy efficiency tests on our 5G platform utilising srsRAN software suite. May you please advise on the price estimate of an AMARI UE Simbox and your experience using it?

**Comment by pgawlowicz** [comment]
Hi, in general, our experience is positive. Please send your question regarding the pricing directly to Amarisoft.
